.github/workflows/dockerhub.yml

```
@@ -0,0 +1,74 @@
+            | sed 's/^[^a-z0-9]\+//; s/[^a-z0-9]\+$//' \
+            | cut -c1-128)
+
+          if [[ -z "$clean" ]]; then
+            clean="branch"
+          fi
+
+          echo "tag=$clean" >> "$GITHUB_OUTPUT"
+          echo "platforms=linux/amd64" >> "$GITHUB_OUTPUT"
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v3
```

actionlint (1.7.9)

the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)